src/index.js

```diff
@@ -136,6 +136,22 @@ async function handleUpdate({
     };
   }
   if (!path) {
+    if (change.uid) {
```

Comment:
why is not having a path a move? in your test, it has a path.

Reply:
Yes, that is somewhat obscure, in function `runPipeline`:

https://github.com/adobe/helix-index-files/blob/11658048421fdaa70d2399a7ef8f0f5bdbba0fcb/src/index.js#L194-L198

when a path is not in a provider's input range, the path is replaced with `null`.

Reply:
but wouldn't it be practical to detect the move ?

Reply:
The move can't be detected because only the index providers can map the `uid` or `sourceHash` to the path the item was moved from.